src/widgets/histogram/content-view.js

```diff
@@ -49,6 +49,7 @@ module.exports = cdb.core.View.extend({
 
     this._renderMiniChart();
     this._renderMainChart();
+    this._renderAllValues();
```

Comment:
I mean, we used to follow the "show_stats" property in order to render the stats or not.